Lab/src/main/java/agh/ics/oop/World.java

```diff
@@ -0,0 +1,174 @@
+package agh.ics.oop;
+import java.util.Arrays;
+import java.util.Objects;
+
+public class World {
+    public static void run(MoveDirection[] directions) {
+        for (MoveDirection direction : directions) {
+            String message = switch (direction) {
+                case FORWARD -> "Zwierzak idzie do przodu";
+                case BACKWARD -> "Zwierzak idzie do tyłu";
+                case LEFT -> "Zwierzak idzie w lewo";
+                case RIGHT -> "Zwierzak idzie w prawo";
+                default -> "Nieznana komenda";
+            };
+            System.out.println(message);
+        }
+    }
+
+    public static MoveDirection[] toEnum(String[] directions) {
+        MoveDirection[] directionsEnum = new MoveDirection[directions.length];
+        for (int i = 0; i < directions.length; i++) {
+            switch (directions[i]) {
+                case "f" -> directionsEnum[i] = MoveDirection.FORWARD;
+                case "b" -> directionsEnum[i] = MoveDirection.BACKWARD;
+                case "l" -> directionsEnum[i] = MoveDirection.LEFT;
+                case "r" -> directionsEnum[i] = MoveDirection.RIGHT;
+                default -> directionsEnum[i] = MoveDirection.UNKNOWN;
+            }
+        }
+        return directionsEnum;
+    }
+
+    public static void main(String[] args) {
+        Animal a = new Animal();
+        OptionsParser o  = new OptionsParser();
+        MoveDirection[] moves = o.parse(new String[]{"f","x","forward","l","eooo","backward"});
+        for (MoveDirection move : moves){
+            a.move(move);
+        }
+        System.out.println((a));
+    }
+}
+
+class Animal {
```

Comment:
nie trzymałbym wielu klas w jednym pliku jeśli nie ma to jakiegoś wyraźnego uzasadnienia